package.json

```diff
@@ -12,5 +11,8 @@
     "hooks": {
       "pre-commit": "pretty-quick --staged"
     }
+  },
+  "dependencies": {
+    "axios": "^0.19.2"
```

Comment:
Axios isn't needed at the top level of this app, this can be removed with `npm uninstall axios` run from root folder.